cpp/open3d/visualization/rendering/filament/FilamentResourceManager.cpp

```diff
@@ -441,10 +456,16 @@ TextureHandle FilamentResourceManager::CreateTexture(const char* path,
 TextureHandle FilamentResourceManager::CreateTexture(
         const std::shared_ptr<geometry::Image>& img, bool srgb) {
     TextureHandle handle;
-    if (img->HasData()) {
+    auto hash = fnv1a_block_hash(img->data_.data(), img->data_.size());
```

Comment:
Are uint8 / float / double images supported? The hash function assumes int data here.

Reply:
The hash function takes a void* and treats the memory block as a block of octets so it should work with any image format.